doc/coding_standards.rst

```diff
@@ -88,17 +88,23 @@ standards:
     [1, 2, 3]
     {'name': 'Fabien'}
 
-* Do not put any spaces before and after ``=`` in macro argument declarations:
+* Put exactly one space before and after ``=`` in macro argument declarations:
```

Comment:
Cf example in https://twig.symfony.com/doc/3.x/tags/macro.html
```
{% macro input(name, value, type = "text", size = 20) %}
```

This seems to be the current standard ; but if you prefer no space, I can update Twig-CS-Fixer.

Reply:
Right, let's keep spaces around `=`.